lms/djangoapps/grades/tasks.py

```diff
@@ -193,6 +195,19 @@ def _has_db_updated_with_new_score_bwc_v2(
     return reported_modified_time >= expected_modified_time
```

Comment:
Nit: It might be useful for debugging to log the actual `reported_modified_time` that was found.